### PR TITLE
Editorial clarification

### PR DESCRIPTION
**Pull request by martinthomson**
It seems like the answer to #13 was already in the draft.  It just wasn't precise enough to be useful.

I also found the wording of the next section awkward, so I'm bundling my changes.

### CODE REVIEW

**Comment by LPardue** [comment]
The editorial improvements seem good but I still don't see an answer about the question of stateless reset. For instance, you mention NEW_TOKEN but not `Stateless Reset Token`.

**Comment by martinthomson** [comment]
Oh, I misread.  Good thing I didn't mark this as closing #13.  I'll add that.

File: draft-ietf-quic-bit-grease.md [hunk 1]
```diff
@@ -90,16 +90,26 @@ current handshake is used to determine what values can be sent in the QUIC Bit.
...
+SHOULD set the QUIC Bit if the token was provided more than 7 days prior.
+
+A server MUST clear the QUIC bit only after processing transport parameters from
+a client.  A server cannot remember that a client negotiated the extension in a
+previous connection and clear the QUIC Bit based on that information.
+
+An endpoint cannot clear the QUIC Bit without knowing whether the peer supports
+the extension.  As stateless reset packets ({{Section 10.3 of QUIC}}) are only
+used after a loss of connection state, endpoints are unlikely to be able to
+clear the QUIC Bit on stateless reset packets.
```
Comment (LPardue) [inline draft-ietf-quic-bit-grease.md:112]:
```suggestion
the extension.  As Stateless Reset packets ({{Section 10.3 of QUIC}}) are only
used after a loss of connection state, endpoints are unlikely to be able to
clear the QUIC Bit on Stateless Reset packets.
```

RFC 9000 uses Capital Casing.

**Review by LPardue — APPROVED** [review]
LGTM thanks

**Pull request merged**